qiskit_experiments/library/heat/elements/zx_elements.py

```diff
@@ -0,0 +1,176 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+HEAT experiment elements for ZX Hamiltonian.
+"""
+
+import numpy as np
+from qiskit import circuit, QuantumCircuit
+
+from qiskit_experiments.library.heat.base_experiment import BaseHeatElement
+
+
+class HeatElementPrepIIMeasIY(BaseHeatElement):
```

Comment:
Make sense. The class structure is simplified in https://github.com/Qiskit/qiskit-experiments/pull/460/commits/cb4d5649e59d20c82bc5be9773e8cbb41e0f1bf2
Basically prep, echo, meas circuits are generated on the fly in the constructor rather than dedicated class, so we don't have naming issue now.

Reply:
I like the simplification.